scene/animation/tween.cpp

```diff
@@ -699,7 +709,9 @@ void PropertyTweener::start() {
 	}
 
 	if (do_continue) {
-		initial_val = target_instance->get_indexed(property);
+		bool access_ok;
+		initial_val = target_instance->get_indexed(property, &access_ok);
+		ERR_FAIL_COND_MSG(!access_ok, "Target property not accessible, aborting Tweener.");
```

Comment:
This is already checked when creating the Tweener (you can't change the property), so no need to check it again.